### ISSUE TITLE
No modification for extention folder using event system

### ISSUE THREAD

**Issue by Samdz19** [opencart/opencart#13255]
**OpenCart 4.0.2.3**

I am having to try to make small modification using event system on report page twig file.

Tried to use startup system, but not working,

for example I use this

```
public function index(): void {
		$this->event->register('extension/opencart/report/customer_order_list/before', new \Opencart\System\Engine\Action('extension/customerid/startup/customer_id.event'));
	}

```

So we use the new ocmod ??

**Comment by pemaier** [comment]
Ocmod is not in the release 4.0.2.3. It is getting implemented in 4.0.2.4 (that's the plan)

**Comment by Samdz19** [comment]
> Ocmod is not in the release 4.0.2.3. It is getting implemented in 4.0.2.4 (that's the plan)

@pemaier yeah, I saw it in 4.0.2.4 but the same issue I didn't got modifications working for extenstions folder.

**Comment by danielkerr** [comment]
this is not true it works. make sure u installed it correctly. the files need to be added from the opencart installer and unzipped. it needs to be unzipped so it can add the extension paths to the templates, library, etc..
